fe/fe-core/src/main/java/com/starrocks/server/LocalMetastore.java

```diff
@@ -826,6 +827,10 @@ public void addPartitions(Database db, String tableName, AddPartitionClause addP
                         "for adding partitions in batches.");
             }
             addPartitions(db, tableName, ImmutableList.of(partitionDesc), addPartitionClause);
+        } else if (partitionDesc instanceof RangePartitionDesc) {
```

Comment:
Batch creation of automatic partition tables that are not at the system level should also be prohibited here.
            if (partitionInfo.getType() == PartitionType.EXPR_RANGE && !partitionDesc.isSystem()) {
                throw new DdlException("Automatically partitioned tables only support the syntax " +
                        "for adding partitions in batches.");
            }

Reply:
fix